### PR TITLE
Update Tooltips for ProjectsView, SamplesView, and ReportTable

### PR DESCRIPTION
**Pull request by j-x-han**
# Description

* Update tooltips for column headers to display the descriptions found [here](https://czi.quip.com/0y0xAXGutQfR/Tooltips-and-Documentation-in-IDseq)
* Tooltips that may require more information will also include a `Learn More` link to documentation found [here](https://github.com/chanzuckerberg/idseq-dag/wiki/IDseq-Pipeline-Stage-%233:-Reporting-and-Visualization)
  * ProjectsView 
Screen Shot 2019-09-10 at 4 23 56 PM
  * SamplesView
Screen Shot 2019-09-10 at 4 25 09 PM
  * ReportTable
Screen Shot 2019-09-10 at 4 25 45 PM

# Notes

* The text for the tooltips taken nearly word-for-word from [here](https://czi.quip.com/0y0xAXGutQfR/Tooltips-and-Documentation-in-IDseq). If you'd like to check the wording, the text can all be found in the `constants.js` files!

* Other small changes made:
  * Column names will turn black when hovered over.
    * If the column is sortable, then the sortIcon `^` will now also be displayed on hover (in addition to when the table is actively being sorted by the column).
  * When hovered over, the icons used to toggle the filters and info sidebars will now turn light blue and display black tooltips saying `Filters` and `Info`, respectively.
Screen Shot 2019-09-10 at 4 28 10 PM
Screen Shot 2019-09-10 at 4 27 29 PM

  * When hovered over, the + icon used to select columns on the SamplesView table will now turn blue, and the tooltip will now display `Add or Remove Columns` rather than `Select Columns`.
Screen Shot 2019-09-10 at 4 26 57 PM

# Tests

* Verified that all column headers (including ones hidden by default on the SamplesView) will display the updated ToolTip.
* Verified that all `Learn More` links go to the proper documentation.

### CODE REVIEW

File: app/assets/src/components/views/report/ReportTable/constants.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  NT_aggregatescore: {
+    title: "Aggregate score",
+    tooltip:
+      "Experimental ranking score for prioritizing microbes based on abundance within the sample (rPM) as well as compared to control samples (Z-score).",
+    link: DOC_LINK,
+  },
+  zscore: {
+    title: "Z-score",
+    tooltip:
+      "Statistic used for evaluating the prevelance of microbes in your sample as compared to background contaminants.",
```
Comment (jshoe) [inline app/assets/src/components/views/report/ReportTable/constants.js]: super-nit: I think we should stick with `the sample` to be consistent (ex https://medium.com/@jsaito/is-this-my-interface-or-yours-b09a7a795256)

File: app/assets/src/components/views/projects/ProjectsView.jsx [hunk 2]
```diff
@@ -21,6 +25,7 @@ class ProjectsView extends React.Component {
 
     this.columns = [
       {
+        columnData: PROJECT_TABLE_COLUMNS["project"],
```
Comment (jshoe) [inline app/assets/src/components/views/projects/ProjectsView.jsx]: For DRY'ing this out, you could instead have a line looping through `this.columns` and adding `columnData: PROJECT_TABLE_COLUMNS[dataKey]`

File: app/assets/src/components/views/samples/SamplesView.jsx [hunk 3]
```diff
@@ -34,31 +35,36 @@ class SamplesView extends React.Component {
 
     this.columns = [
       {
+        columnData: SAMPLE_TABLE_COLUMNS_V2["sample"],
```
Comment (jshoe) [inline app/assets/src/components/views/samples/SamplesView.jsx]: For DRY'ing this out, you could instead have a line looping through `this.columns` and adding `columnData: SAMPLE_TABLE_COLUMNS_V2[dataKey]`

File: app/assets/src/components/views/report/ReportTable/constants.js [hunk 4]
```diff
@@ -0,0 +1,50 @@
+const DOC_LINK =
+  "https://github.com/chanzuckerberg/idseq-dag/wiki/IDseq-Pipeline-Stage-%233:-Reporting-and-Visualization#reports";
```
Comment (jshoe) [inline app/assets/src/components/views/report/ReportTable/constants.js]: just as a reminder, we'll need to make sure these wiki page titles never change or get updated in sync

File: app/assets/src/components/views/samples/constants.js [hunk 5]
```diff
@@ -1,5 +1,72 @@
...
+    subsampled reads to total reads passing host filtration and QC steps.`,
+    link: DOC_LINK,
+  },
+  totalRuntime: {
+    tooltip: `The total time required by the IDseq pipeline to process .fastq files into
+    IDseq reports.`,
+  },
+};
+
+// deprecated
```
Comment (jshoe) [inline app/assets/src/components/views/samples/constants.js]: nit: could you make this all caps

**Comment by jshoe** [comment]
Also a question for @happyimadesignr: should we expect future popups to be on a white background? or only the longer ones?

File: app/assets/src/components/views/discovery/DiscoveryHeader.jsx [hunk 6]
```diff
@@ -73,13 +74,26 @@ class DiscoveryHeader extends React.Component {
...
+                />
+                <Label
+                  className={cs.filtersCounter}
+                  circular
+                  text={`${filterCount}`}
+                />
+              </div>
+            }
+            content="Filters"
+            mouseEnterDelay={500}
```
[the remark this answers is not in the document]
Comment (j-x-han) [reply app/assets/src/components/views/discovery/DiscoveryHeader.jsx]: Oops, good catch, just updated to make them all 500 for consistency. Would like to check with @happyimadesignr to make sure that 500 is a good default though!

**Pull request merged**